README.md

```diff
@@ -10,30 +9,49 @@ README
 What is RollerworksSearch?
 --------------------------
 
-RollerworksSearch provides you with a light yet powerful search system
-to easily integrate into your PHP application.
+RollerworksSearch provides you with a powerful search system
+for your PHP application.
 
 The system has a modular design and can work with any PHP framework,
 user locale, data format or storage system.
 
+Whether you want to simply search for users in your SQL database or want to
+provide a powerful search system for searching products using an ElasticSearch
+back-end, everything is possible.
+
+Say your customer data is stored in the "customer" table while the "invoice"
+data is stored in it's own table. And invoice rows also have there own table.
+
+Searching (using the FilterQuery syntax) can be as simple as:
+`invoice_price: >"$20.00" invoice_row: ~*"my cool product"; customer_type: !consumer.`
```

Comment:
Yes thats one use-case, it abstracts the query building into a search condition which can be used for any back-end storage like SQL or ElasticSearch. Thanks for helping with this :+1: